### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by ahmednawar**
change unit tests to account for deprecating old initSession methods

### CODE REVIEW

**Comment by ahmednawar** [comment]
@aaustin @Sarkar @derrickstaten

**Comment by aaustin** [comment]
:+1: nice

File: Branch-SDK/Branch-SDK/BNCEncodingUtils.m [hunk 1]
```diff
@@ -197,10 +197,10 @@ + (NSString *)iso8601StringFromDate:(NSDate *)date {
...
 + (NSString *)sanitizedStringFromString:(NSString *)dirtyString {
     NSString *cleanString = [[[[dirtyString stringByReplacingOccurrencesOfString:@"\"" withString:@"\\\""]
-                                            stringByReplacingOccurrencesOfString:@"\n" withString:@"\\n"]
-                                            stringByReplacingOccurrencesOfString:@"’" withString:@"'"]
-                                            stringByReplacingOccurrencesOfString:@"\r" withString:@"\\r"];
-
+                               stringByReplacingOccurrencesOfString:@"\n" withString:@"\\n"]
+                              stringByReplacingOccurrencesOfString:@"’" withString:@"'"]
+                             stringByReplacingOccurrencesOfString:@"\r" withString:@"\\r"];
+
```
Comment (Sarkar) [inline Branch-SDK/Branch-SDK/BNCEncodingUtils.m]: minor - do you mind either modifying your editor to be consistent with the whitespace here or unchecking this specific file in? looks like nothing changed

**Comment by Sarkar** [comment]
:+1:

**Pull request merged**